### PR TITLE
feat: add support for swiftpm plugin

### PR DESCRIPTION
**Pull request by tonidevine1**
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/CONTRIBUTING.md) rules

### CODE REVIEW

**Review by DoronElir — APPROVED** [review]

**Review by dotkas — APPROVED** [review]

**Comment by PeterSchafer** [comment]
> Warnings
> ⚠️	
> Looks like you added a new Tap test. Consider making it a Jest test instead. See files in `test/jest/(unit|acceptance)` for examples. Files found:
> 
> * `test/tap/cli-test/cli-test.swift.spec.ts`
> 
> Generated by 🚫 [dangerJS](https://danger.systems/js) against [1d20b70](https://github.com/snyk/cli/commit/1d20b70a30dacb2c248d0e6ad506b645c0728383)

Hi Everyone,
Have you seen the warning about adding TAP tests? We are trying to move test away from TAP to JEST, as we mention in the [contributing guidelines](https://github.com/snyk/cli/blob/master/CONTRIBUTING.md). It would be great if we would move the tests over to jest. 
Thanks.

File: test/tap/cli-test/cli-test.swift.spec.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    '`test swift (autodetect)`': (params, utils) => async (t) => {
+      utils.chdirWorkspaces();
+      const depGraphBuilder = new DepGraphBuilder(
+        { name: 'swift-pm' },
+        { name: 'swift-pm', version: '0.0.1' },
+      );
+      const dependencyGraph = depGraphBuilder.build();
+      const plugin = {
+        async inspect() {
+          console.log('doron is cool');
```
Comment (PeterSchafer) [inline test/tap/cli-test/cli-test.swift.spec.ts:17]: nitpick: :) maybe worth cleaning up? not questioning the statement though.

**Comment by PeterSchafer** [comment]
Suggestion: Could you please squash your commits, to keep the history clean? Thanks!

File: test/tap/cli-test/cli-test.swift.spec.ts (hunk 1 above)
Comment (DoronElir) [reply test/tap/cli-test/cli-test.swift.spec.ts:17]: lol Thanks :)

**Review by PeterSchafer — APPROVED** [review]
@DoronElir and I agreed to have the TAP tests for now and do the JEST tests later.

**Pull request merged**